### PR TITLE
logseq: Update to version 0.1.4

### LINKED ISSUE #6234: Logseq changed the version numbering rule
After [alpha 0.0.20](https://github.com/logseq/logseq/releases/tag/0.0.20) release, Logseq starts Canary release (see https://logseq.github.io/#/page/Canary%20Changelog). The latest version is [0.0.4](https://github.com/logseq/logseq/releases/tag/0.0.4).

Scoop is stuck at 0.0.20. https://github.com/lukesampson/scoop-extras/blob/315f8c194ef3376300f32b5a009ee1cb4a60af3e/bucket/logseq.json

Need manual editing.

### PR DESCRIPTION
**Pull request by woodwm**
. Closes #6234

### CODE REVIEW

**Review by Ash258 — CHANGES_REQUESTED** [review]
- Add autoupdate
- Update to the latest version

File: bucket/logseq.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-    "version": "0.0.20",
+    "version": "0.1.2",
     "description": "A privacy-first platform for knowledge sharing and management",
     "homepage": "https://logseq.com",
     "license": "AGPL-3.0-only",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/logseq/logseq/releases/download/0.0.20/logseq-win-x64-0.0.20.exe#/dl.7z",
-            "hash": "3e943e6886aa71614a856417a3b0843bceba56064a2e4e2059a6d952486cb7eb"
+            "url": "https://github.com/logseq/logseq/releases/download/0.1.2/logseq-win-x64-0.1.2.exe",
```
Comment (Ash258) [inline bucket/logseq.json:8]: Really? Did you even test this before you requested review?

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/logseq.json (hunk 1 above)
Comment (woodwm) [reply bucket/logseq.json:8]: @Ash258 Sorry, I tested checkver and it automatically update the hash and also the link. I did not test install.

**Comment by woodwm** [comment]
@Ash258 I think it should be OK now.  The previous problem is that I did not add the renaming part "#/dl.7z" into autoupdate link. When I tested the autoupdate, it replaced the download link and hash. The renaming part "#/dl.7z" was accidentally removed.

**Pull request merged**